app/templates/index.html

```diff
@@ -11,7 +11,7 @@ <h4>Share your ride</h4>
             <h1>{{ config.get('BRANDING_HEADLINE_1') }}</h1>
             <h2>{{ config.get('BRANDING_HEADLINE_2') }}</h2>
             <form method="GET" action="{{ url_for('carpool.find') }}"">
-                <input type="text" name="q" id="locationQuery" placeholder="Zip code or City, State"> <input class="btn btn-default" type="submit" value="Find a ride">
+                <input type="text" name="q" class="locationQuery" placeholder="Zip code or City, State"> <input class="btn btn-success" type="submit" value="Find a ride">
```

Comment:
There is JavaScript that relies on the `id="locationQuery"` can you undo that change?